app/pb/api.proto

```diff
@@ -385,3 +391,19 @@ message InitiateMediaUploadRes {
   string upload_url = 1;
   google.protobuf.Timestamp expiry = 2;
 }
+
+message UserBlockingReq {
+  int64 user_id = 1;
+}
+
+message BlockedUserRelationship{
+  int64 id = 1;
+  int64 blocking_user_id = 2;
+  int64 blocked_user_id = 3;
+  bool blocked = 4;
+  google.protobuf.Timestamp time_blocked = 5;
+}
+
+message GetBlockedUsersRes {
+  repeated BlockedUserRelationship blocked_user_relationships = 1;
```

Comment:
This can just return a `repeated int64 blocked_user_ids = 1`. The requestor knows their user id, and the time the block was created doesn't need to be returned

Reply:
I considered this, but thought better to make it more versatile for future applications. Happy to minimize it.